src/electron/window-frame-updater.ts

```diff
@@ -2,20 +2,42 @@
 // Licensed under the MIT License.
 import { BrowserWindow } from 'electron';
 import { WindowStateStore } from './flux/store/window-state-store';
-import { WindowStateStoreData } from './flux/types/window-state-store-data';
 
 export class WindowFrameUpdater {
     constructor(private readonly windowStateStore: WindowStateStore, private readonly browserWindow: BrowserWindow) {}
 
     public initialize(): void {
+        this.windowStateStore.addChangedListener(this.onWindowStateChange);
         this.windowStateStore.addChangedListener(this.onRouteChange);
     }
 
-    private onRouteChange = async (state: WindowStateStoreData) => {
-        if (state.routeId === 'deviceConnectView') {
+    private onRouteChange = (store: WindowStateStore) => {
```

Comment:
to avoid this mistake in future, can you update the signature of addChangedListener? let the parameter type be (store: T) => void instead of function

Reply:
I'll take this on a separated PR